### ISSUE TITLE
Hepatitis C Virus Analysis and Prediction

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/ML-Crate#472]
### ML-Crate Repository (Proposing new issue)
:red_circle: **Project Title** : Hepatitis C Virus Analysis and Prediction
:red_circle: **Aim** : Create a analysis and prediction model for the given dataset.
:red_circle: **Dataset** : https://www.kaggle.com/datasets/mohamedzaghloula/hepatitis-c-virus-egyptian-patients
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Participant ID (If not, then put NA) :
- Approach for this Project :
- What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by akanksha-2002** [comment]
Hi, @abhisheks008! I would like to take up this issue. 
Full name: Akanksha Bhimte
GitHub Profile Link: https://github.com/akanksha-2002
Participant ID (If not, then put NA):
Approach for this Project: 

1. EDA ( summary statistics and visualisation of data)
2.  Data Preprocessing ( handling missing values, encoding, feature scaling)
3.  Model implementation ( using LR, KNN, SVM, RF)
4.  Classification metrics to determine which model performs the best. 

What is your participant role? KWOC

**Comment by abhisheks008** [comment]
Issue assigned to you @akanksha-2002

**Comment by vaibhav382** [comment]
Hello @abhisheks008 , could you please assign me this issue under IWOC.

**Comment by vaibhav382** [comment]
Hi, @abhisheks008! I would like to take up this issue.
Full name:                    Vaibhav Pandey
GitHub Profile Link:      https://github.com/vaibhav382
Participant ID :              NA
Approach for this Project:

EDA ( summary statistics and visualisation of data)
Data Preprocessing ( handling missing values, encoding, feature scaling)
Model implementation ( using basic ml algorithms )
Classification metrics to determine which model performs the best.

What is your participant role:  IWOC 2.0

**Comment by abhisheks008** [comment]
**Try to use atleast 3-4 machine learning models for this project.**

Issue assigned to you @vaibhav382

**Comment by vaibhav382** [comment]
@abhisheks008 please unassign me this issue. I am not able to make time for this task. Really sorry for the inconvenience.

**Comment by abhisheks008** [comment]
Sure @vaibhav382

**Comment by aayushraghav93** [comment]
Hi, @abhisheks008! I would like to take up this issue.
Full name: Aayush Raghav
GitHub Profile Link: https://github.com/aayushraghav93
Participant ID : NA
 ## Approach for this Project:-

- Exploratory data analysis
-  Data preprocessing
- Model implementation ( using various ml algorithms such as LR, SVM, RF, )
-  Classification metrics to determine which model performs the best.



What is your participant role: IWOC 2.0

**Comment by abhisheks008** [comment]
Unassigned as the open source event ended up.

**Comment by SiMi723** [comment]
Hi, @abhisheks008! I would like to take up this issue
Full name : Simi
GitHub Profile Link :https://github.com/SiMi723
Participant ID (If not, then put NA) :NA
Approach for this Project : 
EDA ( summary statistics and visualisation of data)
Data Preprocessing ( handling missing values, encoding, feature scaling)
Model implementation ( using basic ml algorithms )
Classification metrics to determine which model performs the best.
What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.): Contributor(SSOC 3.0 & GSSoC)

**Comment by abhisheks008** [comment]
> Hi, @abhisheks008! I would like to take up this issue Full name : Simi GitHub Profile Link :https://github.com/SiMi723 Participant ID (If not, then put NA) :NA Approach for this Project : EDA ( summary statistics and visualisation of data) Data Preprocessing ( handling missing values, encoding, feature scaling) Model implementation ( using basic ml algorithms ) Classification metrics to determine which model performs the best. What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.): Contributor(SSOC 3.0 & GSSoC)

One issue at a time.

**Comment by pratikringe46** [comment]
Hey @abhisheks008,

Can you please assign me this issue under SSOC season 3?
Full Name: Pratik Ringe
[Github](https://github.com/pratikringe46)
Participation ID: NA
Participant Role: SSOC season 3
My approach:
I will be trying 3-4 algos for this: Logistic regression, Naive bayes, SVM, Neural Networks. I have worked on classification and regression models before. The idea would be to implement these model and also provide a comparison between them based on the accuracy and other metrics. I can try using LSTM as well.

Thanks.

**Comment by abhisheks008** [comment]
Implement the following models for this project,
1. Random Forest
2. Decision Tree
3. Logistic Regression
4. Gradient Boosting
5. XGBoost
6. Lasso
7. Ridge
8. MLP Classifier
9. Support Vector Machine


Assigned @pratikringe46
